src/App.tsx

```diff
@@ -16,47 +49,77 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [sortBy, setSortBy] = useState(0);
+  const [isReversed, setIsReversed] = useState(false);
+
+  const reverseToggle = () => {
+    setIsReversed(prev => (!prev));
+  };
+
+  const reset = () => {
+    setIsReversed(false);
+    setSortBy(SortType.NONE);
+  };
+
+  const reorderedGoods = getReorderedGoods(goodsFromServer, {
+    sortType: sortBy,
+    isReversed,
+  });
+
   return (
     <div className="section content">
       <div className="buttons">
         <button
           type="button"
-          className="button is-info is-light"
+          className={classNames('button is-info', {
+            'is-light': sortBy !== SortType.ALPABET,
+          })}
+          onClick={() => {
+            setSortBy(SortType.ALPABET);
+          }}
         >
           Sort alphabetically
         </button>
```

Comment:
some logic for each button is repeated, will be great to find a way to move it to some separate component, that can receive needed props